NEWS.md

```diff
@@ -1,3 +1,7 @@
+# admiral 0.9.1
+
+- Implement changes to `if_else()` from the release of `dplyr` version 1.1.0. (#1641)
```

Comment:
I think we should reference the admiral function that is impacted by this change in dplyr

Reply:
Updated!